src/cli/codecs/farbfeld.rs

```diff
@@ -1,5 +1,5 @@
 use clap::Command;
 
 pub fn farbfeld() -> Command {
-    Command::new("farbfeld").about("Encode images into Farbfeld format")
+    Command::new("farbfeld").about("Encode images into Farbfeld format. (Uncommon)")
```

Comment:
Why do we need `Common` and `Uncommon` remarks? Users that use this app must know about images they want.

Reply:
This is my negligence, i'll correct it